packages/igniter-runtime/lib/-private/promise.ts

```diff
@@ -0,0 +1 @@
+export default window.Promise;
```

Comment:
ah, this file did sneak back in ;)

This should revert to the original implementation.  The TL;DR here is that in production we want to use RSVP.Promise, as RSVP.Promise is still far benchmarking several orders of magnitude faster than native promises on v8 (I last checked this ~4 months ago, is always possible that Chrome has finally caught up).

However, in development we need to ensure use of Native promises if present.  The reasoning is that with the `async` flag set to true in Chrome dev tools, you will get a correct stack trace back to the origin with either RSVP or Native, but Native's is much shorter and much much more legible.